managed-services/transactional-email/how-to/add-dmarc-record.mdx

```diff
@@ -0,0 +1,46 @@
+---
+meta:
+  title: How to configure a DMARC record
+  description: This page explains how to configure a DMARC record
+content:
+  h1: How to configure a DMARC record
+  paragraph: This page explains how to configure a DMARC record
+tags: spf spf-record dkim dkim-records transactional
+dates:
+  validation: 2024-02-21
+  posted: 2024-02-21
+categories:
+  - managed-services
+---
+
+This page shows you how to configure a [DMARC record](/managed-services/transactional-email/concepts/#dmarc-record) on your domain from the **Scaleway console**.
+If you have registered a domain at another provider, you can still use this documentation, as the process is similar.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/).
+  - You have a domain name.
+  - You have [configured your domain with Transactional Email](/managed-services/transactional-email/how-to/configure-domain-with-transactional-email).
+  - You have configured SPF and DKIM records on your domain.
+</Message>
+
+1. Click **Transactional Email** in the **Managed Services** section of the [console](https://console.scaleway.com/) side menu.
+2. Click your domain. You are redirected to the records configuration page.
+3. Add a `TXT` record type in the root zone of your domain on your DNS zone.
+4. Enter `_dmarc` in the **Name** field, followed by the domain name. For example, if your domain is called `my-domain.com`, you should have `_dmarc.my-domain.com.` in the **Name** field.
+5. Enter your desired tags and their values in the **Value** section. Make sure that you add the mandatory `p` and `v` tags, and that you separate tags from each other with semi-colons. Find out about DMARC configuration options in our [dedicated documentation](/managed-services/transactional-email/reference-content/dmarc-configuration/).
+    Your **Value** field should look similar to the following:
+    ```
+    v=DMARC1; p=none; rua=mailto:your@email.com
+    ```
+6. Click **Add record**.
+
+<Message type="tip">
+  For more in-depth information about DMARC configuration, refer to the [dedicated documentation](/managed-services/transactional-email/reference-content/dmarc-configuration/) page.
+</Message>
+
```

Comment:
A note could be added: You can have only one _dmarc TXT records per `Name`
Example: `_dmarc.mydomain.tld` and `_dmarc.tem.mydomain.tld` is valid. but not 2 `_dmarc.mydomain.tld`